drip/views.py

```diff
@@ -1 +1,44 @@
-# Create your views here.
+from typing import Any, Dict, List
+
+from django.views.generic import TemplateView
+
+from drip.tokens import EmailToken
+
+
+class UnsubscribeDripView(TemplateView):
+    template_name = "unsubscribe_drip.html"
+    invalid_template_name = "unsubscribe_drip_invalid.html"
+    success_template_name = "unsubscribe_drip_success.html"
+
+    def _set_user_and_drip(self, **kwargs):
+        drip_uidb64 = kwargs.get("drip_uidb64", "")
+        uidb64 = kwargs.get("uidb64", "")
+        token = kwargs.get("token", "")
```

Comment:
What happens if these values are `""`?

Reply:
It will go to invalid template, but a URL with `///` no data between slashes will break, so I think it is impossible to reach this case.